eventemitter2.d.ts

```diff
@@ -1,6 +1,7 @@
 declare let eventemitter2: eventemitter2.Static
 
 declare module eventemitter2 {
+    type evetns = string[];
```

Comment:
I think the origial author wanted to call it: event namespace, for short, it's ``evetns`` :)
if you think it's not very clear, I can rename it